File: screenshot/Appearance/Geometry.m
//
//  Geometry.m
//  screenshot
//
//  Created by Corey Werner on 8/8/17.
//  Copyright © 2017 crazeapp. All rights reserved.
//

#import "Geometry.h"

@implementation Geometry

+ (CGFloat)padding {
    return 16.f;
}

+ (CGFloat)extendedPadding {
    return 40.f;
}

+ (CGFloat)halfPoint {
    return [UIScreen mainScreen].scale > 1.0f ? 0.5f : 1.0f;
}

@end

File: screenshot/Appearance/Geometry.swift
//
//  Geometry.swift
//  screenshot
//
//  Created by Jacob Relkin on 10/22/17.
//  Copyright © 2017 crazeapp. All rights reserved.
//

import Foundation

extension CGFloat {
    static let padding = CGFloat(16)
    static let extendedPadding = CGFloat(40)
    
    static let halfPoint = CGFloat(UIScreen.main.scale > 1 ? 0.5 : 1)
    
    static let defaultCornerRadius = CGFloat(6)
}

// TODO: remove legacy code after all files are .swift
public class Geometry : NSObject {
    static let padding = CGFloat(16)
    static let extendedPadding = CGFloat(40)
    
    static let halfPoint = CGFloat(UIScreen.main.scale > 1 ? 0.5 : 1)
    
    static let defaultCornerRadius = CGFloat(6)
}
